src/__tests__/__snapshots__/cli_test.js.snap

```diff
@@ -47,15 +47,15 @@ Where:
 
 [1m[32mArguments[39m[22m
 
-    [1m[33m--comments,-c[39m[22m      A string containing comma-separated tokens that
+    [1m[33m--comments,-c[39m[22m      A string containing space-separated tokens that
```

Comment:
Changed to space-separated because some comment types could actually include commas.